roles/openshift_master/defaults/main.yml

```diff
@@ -43,3 +43,89 @@ r_openshift_master_data_dir: "{{ r_openshift_master_data_dir_default }}"
 
 r_openshift_master_sdn_network_plugin_name_default: "{{ os_sdn_network_plugin_name | default('redhat/openshift-ovs-subnet') }}"
 r_openshift_master_sdn_network_plugin_name: "{{ r_openshift_master_sdn_network_plugin_name_default }}"
+
+
+openshift_master_config_dir_default: "{{ openshift.common.config_base ~ '/master' }}"
+openshift_master_config_dir: "{{ openshift_master_config_dir_default | default('/etc/origin/master') }}"
```

Comment:
'| default(...)' should be moved to _default var above.

Reply:
openshift_master_config_dir is set in openshift_master_facts, btw.  We don't need to redefine it here.

Reply:
I would prefer that we use the facts if available but to not hard require them.  I can move the default line up but I don't think facts should be a dependency.